src/thor/dijkstras.cc

```diff
@@ -366,10 +366,16 @@ void Dijkstras::Compute(google::protobuf::RepeatedPtrField<valhalla::Location>&
     // Check if we should stop
     cb_decision = ShouldExpand(graphreader, pred, expansion_direction);
     if (cb_decision != ExpansionRecommendation::prune_expansion) {
-      // Expand from the end node in forward direction.
+      // Expand from the end node in expansion_direction.
       ExpandInner<expansion_direction>(graphreader, pred.endnode(), pred, predindex, opp_pred_edge,
                                        false, time_infos.front());
     }
+
+    if (expansion_callback_) {
+      // "algorithm" arg defaults already to dijkstra
+      expansion_callback_(graphreader, pred.edgeid(), nullptr, "s", pred.cost().secs,
```

Comment:
i see the comment but still, i couldnt see where it defaults in the code. i think it would also be odd to use a default, do you mind  if we just force the caller to specify the algorithm?

Reply:
it's odd I agree. I'll change that